app/javascript/packages/document-capture/context/in-person.ts

```diff
@@ -36,6 +36,17 @@ export interface InPersonContextProps {
    * Each item is [Long name, abbreviation], e.g. ['Ohio', 'OH']
    */
   usStatesTerritories: Array<[string, string]>;
+
+  /**
+   * When true users are directed to the beginning of the IPP flow if they choose
+   * Opt-in IPP if in_person_proofing_opt_in_enabled is enabled
```

Comment:
I had to read this a few times, so I'm suggesting clarification.
```suggestion
   * When skipDocAuth is true and in_person_proofing_opt_in_enabled is true,
   * users are directed to the beginning of the IPP flow. This is set to true when
   * they choose Opt-in IPP on the new How To Verify page
```

Reply:
Done. Resolved with commit 54cc97ab425b492b38897748bc91d4823488f5e8 

(I did not commit- I wanted to run linter before pushing up)